feature/userinfo/src/main/java/com/squirtles/feature/userinfo/screen/EditPlayerScreen.kt

```diff
@@ -0,0 +1,225 @@
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.selection.selectable
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.RadioButton
+import androidx.compose.material3.RadioButtonDefaults
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.squirtles.core.common.ui.Constants.COLOR_STOPS
+import com.squirtles.core.common.ui.DefaultTopAppBar
+import com.squirtles.core.common.ui.theme.Primary
+import com.squirtles.core.common.ui.theme.White
+import com.squirtles.core.preference.PreferenceViewModel
+import com.squirtles.domain.preference.PlayerPreference
+import com.squirtles.feature.userinfo.R
+
+@Composable
+fun EditPlayerScreen(
+    modifier: Modifier = Modifier,
+    onBackClick: () -> Unit,
+    preferenceViewModel: PreferenceViewModel = hiltViewModel()
+) {
+    val currentEffect by preferenceViewModel.playerPreference.collectAsStateWithLifecycle(null)
+
+    val effectNameMap = mapOf(
+        PlayerPreference.NONE to stringResource(R.string.sound_effect_none),
+        PlayerPreference.BAR to stringResource(R.string.sound_effect_bar),
+        PlayerPreference.STROKE to stringResource(R.string.sound_effect_wave_stroke),
+        PlayerPreference.FILL to stringResource(R.string.sound_effect_wave_fill)
+    )
+
+    val savePreference: (PlayerPreference) -> Unit = {
+        preferenceViewModel.savePlayerPreference(it)
+    }
+    
+    Scaffold(
+        topBar = {
+            DefaultTopAppBar(
+                title = "플레이어 설정",
+                onBackClick = onBackClick
+            )
+        }
+    ) { innerPadding ->
+        Box(
+            modifier = Modifier
+                .fillMaxSize()
+                .background(Brush.verticalGradient(colorStops = COLOR_STOPS))
+                .padding(innerPadding),
+        ) {
+            if(currentEffect != null){
```

Comment:
코드 자동 정렬이 안되어 있는 것 같습니다 ㅎㅎ 
```suggestion
            if (currentEffect != null) {
```